tests/perf/perf_tests.cc

```diff
@@ -160,7 +167,6 @@ struct result {
     double inst = 0.;
 };
 
-namespace {
```

Comment:
Please, re-split the series. First patch is to de-anonymize the duration class. Second patch to add formatters for the missing pieces.

Reply:
@xemul Please take a look again